src/components/utils/DeleteButton.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @author aramsey
+ *
+ * A commonly used button or icon button that displays a red trash can icon
+ */
+import React from "react";
+
+import { build } from "@cyverse-de/ui-lib";
+import { Delete } from "@material-ui/icons";
+import { Button, IconButton } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+import ids from "./ids";
+import { makeStyles } from "@material-ui/styles";
+import { useTranslation } from "../../i18n";
+
+const useStyles = makeStyles((theme) => ({
+    deleteBtn: {
+        color: theme.palette.error.main,
+    },
```

Comment:
Forgot to mention this before, but in the app editor I had to use `backgroundColor: theme.palette.error.contrastText` for its delete icons, since the red on a dark background (such as in a groups accordion summary) doesn't stand out well:
![App Editor group action icons](https://user-images.githubusercontent.com/996408/114634089-a7c96600-9c76-11eb-9f90-f33de31f05b0.png)

I'm wondering if we should also add this background contrast color to this common delete button in a future PR.